nixos/modules/services/web-servers/apache-httpd/default.nix

```diff
@@ -724,14 +728,15 @@ in
           ExecStart = "@${pkg}/bin/httpd httpd -f ${httpdConf}";
```

Comment:
Don`t need change to 
```ExecStart = "${pkg}/bin/httpd -f ${httpdConf}";```

Reply:
Why change to that?

Reply:
To run without root privileges

Reply:
Ah... yes, easy mistake to make. You were thinking of `+`, not `@`.

https://www.freedesktop.org/software/systemd/man/systemd.service.html
> If the executable path is prefixed with "@", the second specified token will be passed as "argv[0]" to the executed process (instead of the actual filename), followed by the further arguments specified.

Not running as `root` :+1:

Reply:
Nginx correct worked without "@"

```
      execCommand = "${cfg.package}/bin/nginx -c '${configPath}' -p '${cfg.stateDir}'";
      ...
      serviceConfig = {
        ExecStart = execCommand;
```